kubernetes/example.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
```

Comment:
can you put the code in a `ifmain` block, and add `#!/usr/bin/env python` to the top so that this code is easily executable?

Reply:
I'll put in the main block, but I always runs scripts with `python script.py`. What's the advantage to allowing `./script.py`?

Reply:
I think the costs outweigh the benefits. Especially since it can find the wrong version of python (we don’t know which python executable the user wants to use).